pkg/controller/control.go

```diff
@@ -759,6 +765,31 @@ func (c *Controller) WriteAt(b []byte, off int64) (int, error) {
 	return n, err
 }
 
+func (c *Controller) writeInWOMode(b []byte, off int64) (int, error) {
+	sectorSize := int64(4096)
```

Comment:
use `defaultSectorSize` const for this value instead

Reply:
Thank you. Fixed